docs/scratch_orgs.rst

```diff
@@ -1,38 +1,253 @@
+* Scratch orgs are scalable and ensure that individual, customized environments are available to everyone in the development lifecycle.
+* Scratch orgs facilitate a fully source-driven development process built around best practices.
+
+CumulusCI offers tools for working with all types of Salesforce orgs, but provides the most value when working with scratch orgs. CumulusCI automation helps realize the promise of scratch orgs as low cost, repeatable, source-driven environments for every phase of the product lifecycle.
+
+This section focuses on managing scratch orgs in a CumulusCI project. To learn about managing persistent orgs, such as sandboxes, production orgs, and packaging orgs, read `Connect persistent orgs<TODO>`_.
+
+
+
+What Is an Org in CumulusCI?
+----------------------------
+
+An org in CumulusCI's keychain starts out as a named configuration, tailored for a specific purpose within the lifecycle of the project (such as development, feature, beta, and so on). CumulusCI creates and uses scratch orgs that are portable and repeatable. In fact, a scratch org is only generated the first time you use the scratch org. When it's expired or been deleted, a new one can easily be created again with the same configuration file.
+
+CumulusCI offers tools that make it easy to discover predefined org configurations, create scratch orgs based on those configurations, and define new orgs and new configurations.
+
+
+
+Set Up SFDX 
+-----------
+
+Scratch orgs in CumulusCI allow teams to be confident that the orgs they develop and test in are as close to their production environments as possible. We recommend working with scratch orgs created by Salesforce DX.
+
+.. note:: For a detailed introduction to set up Salesforce CLI and Visual Studio Code to work with CumulusCI, we recommend completing the `Build Applications with CumulusCI <https://trailhead.salesforce.com/en/content/learn/trails/build-applications-with-cumulusci>`_ module on Trailhead.
 
-Predefined orgs
+To set up Salesforce DX:
+
+1. `Install Salesforce CLI <https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm>`_
+2. `Enable Dev Hub Features in Your Org <https://developer.salesforce.com/docs/atlas.en-us.228.0.sfdx_dev.meta/sfdx_dev/sfdx_setup_enable_devhub.htm>`_
+3. `Connect SFDX to Your Dev Hub Org <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_web_flow.htm>`_ 
+
+.. important :: Be sure to use the ``--setdefaultdevhubusername`` option!
+
+If you have the ``sfdx`` command installed, are connected to your DevHub, and set the ``defaultdevhubusername`` config setting (use the ``sfdx force:config:list`` command to verify!), you're now ready to use ``cci`` with ``sfdx``.
+
+.. note:: SFDX supports multiple DevHubs, so CumulusCI uses the one set as ``defaultdevhubusername`` when creating scratch orgs.
+
+Learn more about Salesforce DX at https://developer.salesforce.com/platform/dx.
```

Comment:
I think we should avoid having raw links in the documentation. I can't think of a great way to incorporate this. Maybe something like:
```suggestion
.. tip::
    The Salesforce Developers site has comprehensive documentation regarding `Salesforce DX <https://developer.salesforce.com/platform/dx.>`_.
```